ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -796,6 +800,16 @@ export enum AuthenticationKeyScheme {
    * For MultiEd25519PublicKey
    */
   MultiEd25519 = 1,
+  /**
+   * For Secp256k1 ecdsa
+   */
+  Secp256k1Ecdsa = 2,
```

Comment:
where are the numbers coming from? is it set in the rust code? if so lets add a link to it, same as https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk_v2/src/types/index.ts#L58

Reply:
Yep its defined on the rust side. Will do!